backend/api_calls.py

```diff
@@ -728,6 +728,64 @@ def AddPlayerToChat(request, firebase):
   results.append(firebase.put('/games/%s/chatRooms' % game, chat, True))
 
 
+def Infect(request, firebase):
+  """Infect a player(otherPlayerId) and update access to chat rooms accordingly.
+
+  Validation:
+    playerId:
+    otherPlayerId:
+    gameId:
+
+    
+  """
+  results = []
+  valid_args = ['infectionId', 'playerId', 'otherPlayerId', 'gameId']
```

Comment:
Ability to infect via life code was to be added in a later push due to having to map life codes to players, but it can happen here, too